src/Contexts/dataProvider.js

```diff
@@ -0,0 +1,91 @@
+import React, { createContext, useContext, useReducer, useState } from "react";
+import { checkIfProductExistsInList } from "../util";
+import { dataReducer } from "./dataReducer";
+
+export const DataContext = createContext();
+export function useData() {
+  return useContext(DataContext);
+}
+
+export function DataProvider({ children }) {
+  const [open, setOpen] = useState();
+  const [state, dispatch] = useReducer(dataReducer, {
+    products: [],
+    wishListItems: [],
+    cartItems: [],
+    searchedItems: [],
+    sortBy: null,
+    showInventoryAll: true,
+    showFastDeliveryOnly: false,
+    snackbarText: null,
+    showLoginModal: false,
+    petsType: null
+  });
+
+  function addToCart(productItem) {
+    dispatch({ type: "OPEN_SNACKBAR", payload: "Added to cart !" });
+    console.log("isaddedtoCart", productItem.isAddedToCart);
+    !checkIfProductExistsInList(state.cartItems, productItem.id)
+      ? dispatch({
+          type: "ADD_ITEM_TO_CART",
+          payload: { ...productItem, quantity: 1 }
+        })
+      : dispatch({
+          type: "INCREMENT_QUANTITY",
+          payload: productItem
+        });
+    console.log("isaddedtoCart", productItem.isAddedToCart);
```

Comment:
Couldn't find anything relevant.
So suggesting to remove console logs

Reply:
Yes I have removed console logs from everywhere now. Thanks.